Makefile

```diff
@@ -726,6 +728,14 @@ bazel-protokube-export:
 	cp -fp bazel-bin/images/protokube.tar.gz.sha1 ${BAZELIMAGES}/protokube.tar.gz.sha1
 	cp -fp bazel-bin/images/protokube.tar.gz.sha256 ${BAZELIMAGES}/protokube.tar.gz.sha256
 
+.PHONY: bazel-kube-apiserver-healthcheck-export
+bazel-kube-apiserver-healthcheck-export:
+	mkdir -p ${BAZELIMAGES}
+	DOCKER_REGISTRY="" DOCKER_IMAGE_PREFIX="kope/" KUBE_APISERVER_HEALTHCHECK_TAG=${KUBE_APISERVER_HEALTHCHECK_TAG} bazel build ${BAZEL_CONFIG} --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //cmd/kube-apiserver-healthcheck:image-bundle.tar.gz //cmd/kube-apiserver-healthcheck:image-bundle.tar.gz.sha1 //cmd/kube-apiserver-healthcheck:image-bundle.tar.gz.sha256
+	cp -fp bazel-bin/cmd/kube-apiserver-healthcheck/image-bundle.tar.gz ${BAZELIMAGES}/kube-apiserver-healthcheck.tar.gz
+	cp -fp bazel-bin/cmd/kube-apiserver-healthcheck/image-bundle.tar.gz.sha1 ${BAZELIMAGES}/kube-apiserver-healthcheck.tar.gz.sha1
```

Comment:
Why do we need to create and ship SHA-1 hashes?

Reply:
I believe it's historical and we began migrating to sha256 about a year ago.  I'm not sure on the specifics, we can probably talk about a deprecation at some point, however I don't think this PR is the right time to do it.

Reply:
Good call - it's a new file, let's try removing the sha1, then we can find out if anyone is still using it (they shouldn't be!)

Reply:
Oops - our responses crossed.  The sha256 support (eca2ac6b80a2d2cb269e4494d3fa7cf23cba8a6c) should be in kops 1.15, so I am going ahead and removing the sha1.  But if it gives us trouble, let's add it back ...